clang/lib/CodeGen/CodeGenTypeCache.h

```diff
@@ -51,14 +51,11 @@ struct CodeGenTypeCache {
     llvm::IntegerType *PtrDiffTy;
   };
 
-  /// void* in address space 0
+  /// void*, void** in address space 0
   union {
+    llvm::PointerType *UnqualPtrTy;
```

Comment:
Maybe `OpaquePtrTy` is a better name. But `Unqual` seem to be used in other places and I've typically used this instead of `PointerType::getUnqual`. But I'm not sure what `unqual` has been denoting originally (a pointer in address space zero?).

Reply:
Not qualified (c.f. QualType)

Reply:
(Presumably the IR type's nomenclature has its origins in C's language)

Reply:
Ok , thanks. Then I assume the name is fine.